submissions/bunmiOladipupo/react-app/public/index.html

```diff
@@ -27,6 +27,50 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
     <link rel="stylesheet" href="%PUBLIC_URL%/../../src/index.css">
     <title>React App</title>
+    <style>
+        #verify{
+            margin-bottom: 20px;
+        }
+        #deliveryorder{
+            font-size: 10px;
+            width: 100%;
+        }
+        #homecart{
+            margin: 30px 0px 20px;
+        }
+        .deliveryaddress{
+          margin: 20px 0px;
+        }
+        hr{
+          margin: 30px 0px;
+        }
+        label{
+          margin: 20px 0px;
+        }
+        #radio{
+          color: red;
+          background-color: red;
+        }
+        input[type="radio"]{
+          accent-color: red;
+          width: 60% !important;
+        }
+        #inputBlock{
+          width: 100% !important;
+        }
+        @media screen and (min-width: 760px){
+          #deliveryorder{
+            font-size: 20px;
+            width: 100%;
+          }
+          #deliver{
+            font-size: 24px;
+          }
+          #radio{
+            margin-left: 50px;
+          }
+        }
+    </style>
```

Comment:
Hello @Bunmi194 Not sure why you decided insert you css styling here. It's more professional to separate your html tags from your css styling.